sdk/keyvault/keyvault-keys/README.md

```diff
@@ -40,6 +40,18 @@ To quickly create the needed Key Vault resources in Azure and to receive a conne
 
 [![](http://azuredeploy.net/deploybutton.png)](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-sdk-for-js%2Fmaster%2Fsdk%2Fkeyvault%2Ftest-resources.json)
 
+To fill this form:
+
+1. If you know about ARM templates, or if you're interested to know about them, we recommend you to take some time to read through the ARM template we are using for this button [here](https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-sdk-for-js%2Fmaster%2Fsdk%2Fkeyvault%2Ftest-resources.json). You can read more about the concept of an ARM template in our [Intro to Microsoft Azure Resource Manager Templates (Video)](https://dev.to/azure/intro-to-microsoft-azure-resource-manager-templates-video-9cc).
```

Comment:
the first line feels like it should be some text before the list instead of the first item, but I know you want people to actually read it...

Reply:
I'm open to suggestions! This is meant as a conversation starter 🌞